src/components/CurrentDirectors/CurrentDirectors.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div id="current-directors">
+
+    <v-card flat>
+      <!-- Header -->
+      <div class="header-container">
+        <v-icon color="app-dk-blue">mdi-account-multiple-plus</v-icon>
+        <label class="font-weight-bold pl-2">Current Directors</label>
+      </div>
+
+      <!-- Instructional Text -->
+      <div class="instructional-text pt-10 pb-15 px-4">
+        To change directors, please use the Change feature in the Current Directors list on you business dashboard.
+      </div>
+
+      <div class="list-container px-4">
+        <!-- List Display Section -->
+        <div id="people-roles-list" v-if="currentDirectors.length > 0">
```

Comment:
No need to change this atm, but if you have a code section, consider using the HTML5 semantic elements (eg, "section").

https://www.google.com/search?q=html5+semantic+elements

This helps with accessibility, as various tools parse the document and can identify headers/sections/footers/etc to the user.

Basically, if you see an obvious opportunity to use semantic elements instead of divs, go for it.